library/core-components/components/auto-layout/auto-layout.types.ts

```diff
@@ -69,7 +69,7 @@ export type AutoLayoutExtendedProps = {
   /** Used in conjunction with isParent, uses and sets position: absolute */
   absolutePosition?: boolean;
   /** The direction of the layout, uses flex row or column */
-  direction?: 'horizontal' | 'vertical';
+  direction?: CSSProp<'other', 'component'>;
```

Comment:
One interesting side effect of using other though is that it isn’t very type safe. Since any properties under other will basically have the same type.

See slack discussion here: https://rangle.slack.com/archives/C04DKV4CZ46/p1683127339978339